azure-keyvault-secret-spring-boot/src/test/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+azure.keyvault.client-id=testid
+azure.keyvault.client-key=testkey
+azure.keyvault.uri=fakeuri
+azure.keyvault.enabled=false
```

Comment:
I think the test would be more meaningful if we can verify property source `azurekv` exists. Or you can add two test cases here.